nuget.config

```diff
@@ -7,5 +7,6 @@
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="particular myget.org" value="https://www.myget.org/F/particular/" />
+	<add key="particular community myget.org" value="https://www.myget.org/F/particular-community/" />
```

Comment:
This affects all doco projects, and [the feed itself appears to be empty](https://www.myget.org/feed/Packages/particular-community)? So what's up?